packages/hint-manifest-app-name/src/hint.ts

```diff
@@ -63,7 +60,7 @@ export default class ManifestAppNameHint implements IHint {
         };
 
         const validate = async (manifestParsed: ManifestParsed) => {
-            const { parsedContent: manifest, resource }: { parsedContent: Manifest, resource: string } = manifestParsed;
+            const { getLocation, parsedContent: manifest, resource } = manifestParsed;
```

Comment:
Can you please add the types back? We are trying to go full strict (#576) and I think this is needed.

Reply:
Really? TypeScript fully infers those types correctly so I'm surprised that they'd be required... 

I might try a local test case first just to make sure this is necessary in full strict mode - I hate unnecessarily redundant type definitions. :)

Reply:
We've had some problems in the past believing we were getting one type when it was actually another, or when we changed the return type and nothing complained and then hell happen...